### PR TITLE
Separate validations

### PR DESCRIPTION
**Pull request by Tristramg**

### CODE REVIEW

File: lib/transport_web/controllers/resource_controller.ex [hunk 1]
```diff
@@ -1,28 +1,29 @@
 defmodule TransportWeb.ResourceController do
   use TransportWeb, :controller
-  alias Transport.{Repo, Resource}
+  alias Transport.{Repo, Resource, Validation}
 
   def details(conn, params) do
     config = make_pagination_config(params)
     id = params["id"]
 
+
```
Comment (l-vincent-l) [inline lib/transport_web/controllers/resource_controller.ex]: useless line break

File: lib/transport/resource.ex [hunk 2]
```diff
@@ -80,10 +79,15 @@ defmodule Transport.Resource do
...
 
     __MODULE__
+    |> preload(:validation)
     |> Repo.get(id)
-    |> change(validation_date: DateTime.utc_now |> DateTime.to_string)
-    |> change(validations: validations)
-    |> change(metadata: metadata)
+    |> change(
+      metadata: metadata,
+      validation: %Validation{
```
Comment (l-vincent-l) [inline lib/transport/resource.ex]: I think you may want to write `validations`

Comment (l-vincent-l) [inline lib/transport/resource.ex]: I think you may want to write validadtions

File: lib/transport/validation.ex [hunk 3]
```diff
@@ -0,0 +1,14 @@
+defmodule Transport.Validation do
+  @moduledoc """
+  Validation model
+  """
+  use Ecto.Schema
+  alias Transport.{Resource, Repo}
+
+  schema "validations" do
+    field :validations, :map
```
Comment (l-vincent-l) [inline lib/transport/validation.ex]: I would prefer a name like `details`

Comment (Tristramg) [reply lib/transport/validation.ex]: done

**Pull request merged**